lld/COFF/Options.td

```diff
@@ -79,6 +79,9 @@ def lldltocachepolicy : P<"lldltocachepolicy",
     "Pruning policy for the ThinLTO cache">;
 def lldsavetemps : F<"lldsavetemps">,
     HelpText<"Save intermediate LTO compilation results">;
+def lldsavetemps_eq : Joined<["/", "-", "/?", "-?"], "lldsavetemps:">,
```

Comment:
Small nitpick - the `_eq` suffix feels a little bit off here, as it's not a case of `=` but actually `:`. Not sure if I have anything better to suggest though.

Reply:
It seems `_colon` is used for `manifest_colon`. It probably makes sense to use `_colon` here as well.

Reply:
I've updated to use `_colon`.